test/e2e/storage/nfs_persistent_volume-disruptive.go

```diff
@@ -88,20 +88,18 @@ var _ = utils.SIGDescribe("NFSPersistentVolumes", framework.WithDisruptive(), fu
 			Selector:         selector,
 			StorageClassName: &emptyStorageClass,
 		}
-		// Get the first ready node IP that is not hosting the NFS pod.
-		if clientNodeIP == "" {
+		if clientNode == nil {
 			framework.Logf("Designating test node")
 			nodes, err := e2enode.GetReadySchedulableNodes(ctx, c)
 			framework.ExpectNoError(err)
 			for _, node := range nodes.Items {
 				if node.Name != nfsServerPod.Spec.NodeName {
 					clientNode = &node
-					clientNodeIP, err = e2enode.GetSSHExternalIP(clientNode)
```

Comment:
Hmm, I think we need to check the node IP.
cc @andyzhangx

Reply:
@xing-yang we don't use this anywhere...

Reply:
We do fallback to internal address if external address is not available - https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/utils/pod.go#L164

So we should be fine.